### ISSUE TITLE
Verify Sort: ct_data_loader.py

### ISSUE THREAD

**Issue by Neurobiologist** [Neurobiologist/EC601-Pulmonary-Embolism#16]
**ct_data_loader.py:**

    """ Base class DataLoader for loading a 3d dataset. This data loader is designed to work with
    sequential models, and takes care of sorting batches and padding them for the pytorch
    recurrent networks. Note that the dataset MUST BE SORTED BY LENGTH for this to work."""

_Originally posted by @Neurobiologist in https://github.com/Neurobiologist/EC601-Pulmonary-Embolism/issues/14#issuecomment-716634405_
